### PR TITLE
Readme: Install npm package globally

### PR DESCRIPTION
**Pull request by bencevans**

### CODE REVIEW

**Comment by cjb** [comment]
@bencevans Thanks!  Perhaps this should be `sudo npm install -g gittorrent`?

**Comment by bencevans** [comment]
In README's I'd suggest longform options (--global rather than -g) as 
for users that arn't used to npm etc might get a better idea.

Ideally sudo shouldn't be used when installing modules either. Stick 
them in userland! :D

**Comment by cjb** [comment]
Ah, that's fair.  But won't it just fail for everyone by trying to write to /usr/local, when you use --global but not sudo?

**Comment by ralphtheninja** [comment]
@cjb It depends on how you have setup your system. Personally I have setup so my current user owns `/usr/local`

**Comment by cjb** [comment]
But the point of the README is to explain how to install npm modules to people who don't already know, and those people are very likely not to have set up their system to make --global work without sudo.

It seems like it would make more sense to include sudo in the instructions, and then people who've done as @ralphtheninja did will know that it's safe for them to remove the sudo.

**Comment by ralphtheninja** [comment]
Or perhaps write something like

```
$ [sudo] npm i -g gittorrent
```

I'd say the convention is to just write `npm i -g xyz` to inform the user that it's a global install and leave up the user to decide if he/she needs to do sudo or not.

**Comment by swizzard** [comment]
Why not

    add the --global flag to install globally (may require superuser privileges)

**Comment by ralphtheninja** [comment]
If this is only meant as a cli tool it should always be installed globally imo.

@cjb Or is this meant to be used as a dependency as well?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 To get started:
 ```
-npm install gittorrent
+npm install --global gittorrent
```
Comment (5807370236) [inline README.md]: 9

**Comment by cjb** [comment]
@bencevans Thanks for the patch!  I'll merge this, add a sudo, and add a note explaining that you can avoid using sudo if you get the binaries into your $PATH.

**Comment by ralphtheninja** [comment]
:+1:

**Comment by bencevans** [comment]
Sweet!

**Pull request merged**